src/WebJobs.Script/Host/DistributedLockManagers/BlobLeaseDistributedLockManager.cs

```diff
@@ -0,0 +1,383 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Diagnostics;
+using System.Globalization;
+using System.IO;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure;
+using Azure.Storage.Blobs;
+using Azure.Storage.Blobs.Models;
+using Azure.Storage.Blobs.Specialized;
+using Microsoft.Azure.WebJobs.Host;
+using Microsoft.Azure.WebJobs.Logging;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Azure.WebJobs.Script
+{
+    /// <summary>
+    /// This is an overriden implementation based off the StorageBaseDistributedLockManager in Microsoft.Azure.WebJobs.Host.Storage package.
```

Comment:
In a remarks section here, can we lable this TEMP to make it clear that this is a stop-gap solution until we release the actual storage v12 version of Microsoft.Azure.WebJobs.Host.Storage, when this will go away?